kubernetes/configs/prod/versions.sh

```diff
@@ -19,10 +19,10 @@ TENANT_PROCESS_IMAGE_VERSION="1.0.8"
 API_GATEWAY_IMAGE_VERSION="1.0.15"
 BACKEND_FOR_FRONTEND_IMAGE_VERSION="1.0.12"
 NOTIFIER_IMAGE_VERSION="1.0.10"
+CERTIFIED_MAIL_SENDER_IMAGE_VERSION="1.0.15"
```

Comment:
probably it is 16 now
```suggestion
CERTIFIED_MAIL_SENDER_IMAGE_VERSION="1.0.16"
```

Since this follows the same tags of the jobs, it might be helpful to move this line right before the jobs, so we do not forget to bump it

Reply:
I suggest waiting for this update because probably we need to set version 1.0.18 here (this version does not exist but it will contain the renaming of env vars, [see](https://github.com/pagopa/interop-be-jobs/pull/130))

Reply:
> I suggest waiting for this update because probably we need to set version 1.0.18 here (this version does not exist but it will contain the renaming of env vars, [see](https://github.com/pagopa/interop-be-jobs/pull/130))

I see 1.0.17 on that repo, do we wait for the next release? @beetlecrunch